File: web/demo/npm/src/models/Transaction.ts
export interface Transaction {
    timestamp: string;
    transaction_id: string;
    originating_wallet: string;
    originating_vasp_id: string;
    originating_vasp_display_name: string;
    beneficiary_wallet: string;
    beneficiary_vasp_id: string;
    beneficiary_vasp_display_name: string;
    ivms101Data: string;
}

File: web/demo/bff/flaskr/models/transaction.py
from dataclasses import dataclass

from dataclasses_json import dataclass_json

from flaskr.models.vasp_details import VaspDetails


@dataclass_json
@dataclass
class Transaction:
    timestamp: str
    transaction_id: str
    originating_wallet: str
    originating_vasp_id: str
    originating_vasp_display_name: str
    beneficiary_wallet: str
    beneficiary_vasp_id: str
    beneficiary_vasp_display_name: str
    ivms101Data: str
